src/components/Property/types.ts

```diff
@@ -1,6 +1,19 @@
 // These props are the same for all property types
-export interface ConcretePropertyBaseProps {
-  name: string;
-  description: string;
-  disabled: boolean;
+export interface PropertyProps {
+  uri: string;
+  readOnly: boolean;
+}
+
+export interface AdditionalDataNumber {
+  Exponent: number;
+  MaximumValue: number;
+  MinimumValue: number;
+  SteppingValue: number;
+}
+
+export interface AdditionalDataVectorMatrix {
```

Comment:
Move this to VectorProperty.tsx and export it from there?

Reply:
We shouldn't export interfaces from .tsx files though, so I would rather see us move all the other `AdditionalData` types here instead